lib/static/new-ui/features/suites/components/SuitesPage/index.tsx

```diff
@@ -76,57 +77,70 @@ function SuitesPageInternal({currentResult, actions, treeNodeId}: SuitesPageProp
-                <ErrorHandler.Boundary watchFor={[currentResult, suiteIdParam, isInitialized]} fallback={<ErrorHandler.FallbackCardCrash recommendedAction={'Try to choose another item'}/>}>
-                    {currentResult && <>
-                        <div className={styles.stickyHeader}>
-                            <SuiteTitle
-                                className={styles['card__title']}
-                                suitePath={currentResult.suitePath}
-                                browserName={currentResult.name}
-                                index={currentIndex}
-                                totalItems={visibleTreeNodeIds.length}
-                                onNext={(): void => onPrevNextSuiteHandler(1)}
-                                onPrevious={(): void => onPrevNextSuiteHandler(-1)} />
-                            <AttemptPicker onChange={onAttemptChangeHandler} />
-                        </div>
-
-                        <CollapsibleSection title={'Overview'} id={'overview'} className={styles['collapsible-section-overview']} body={
-                            currentResult &&
-                            <div className={styles['collapsible-section__body']}>
-                                <ErrorHandler.Boundary watchFor={[currentResult]} fallback={<ErrorHandler.FallbackDataCorruption />}>
-                                    <MetaInfo resultId={currentResult.id} />
-                                </ErrorHandler.Boundary>
-                            </div>
-                        }/>
-
-                        <ErrorHandler.Boundary watchFor={[currentResult]} fallback={<ErrorHandler.FallbackDataCorruption />}>
-                            <TestSteps />
-                        </ErrorHandler.Boundary>
-                    </>}
+    const isTimeTravelEnabled = useSelector(state => state.app.availableFeatures.some(f => f.name === TimeTravelFeature.name));
 
-                    {!suiteIdParam && !currentResult && <div className={styles.hintContainer}><span className={styles.hint}>Select a test to see details</span></div>}
-                    {suiteIdParam && !isInitialized && <TestInfoSkeleton />}
-                </ErrorHandler.Boundary>
-            </UiCard>
-        ]} />
-    </div>;
+    const [stickyHeaderElement, setStickyHeaderElement] = useState<HTMLDivElement | null>(null);
+    const [stickyHeaderHeight, setStickyHeaderHeight] = useState(0);
+    useLayoutEffect(() => {
+        if (!stickyHeaderElement) {
+            return;
+        }
+        setStickyHeaderHeight(stickyHeaderElement.clientHeight);
+        const resizeObserver = new ResizeObserver((entries) => {
+            setStickyHeaderHeight(entries[0].contentRect.height);
+        });
+        resizeObserver.observe(stickyHeaderElement);
+
+        return () => resizeObserver.disconnect();
+    }, [stickyHeaderElement]);
+
+    return <div className={styles.container}><SplitViewLayout>
```

Comment:
Here, nothing changed besides adding time travel section if experiment is enabled. The rest of the diff is due to moving from `sections={[]}` API to children in SplitView.